content/en/guides/models/track/log/distributed-training.md

```diff
@@ -100,6 +100,52 @@ Explore the W&B App UI to view an [example dashboard](https://wandb.ai/ayush-tha
 
 The preceding image demonstrates the W&B App UI dashboard. On the sidebar we see two experiments. One labeled 'null' and a second (bound by a yellow box) called 'DPP'. If you expand the group (select the Group dropdown) you will see the W&B Runs that are associated to that experiment.
 
+### Organize distributed runs with `job_type`
+
+When running distributed training with multiple processes, you can specify the `job_type` in your [`wandb_init()`]({{< relref "/ref/python/sdk/functions/init/" >}}) invocation to distinguish between different types of nodes. Then you can create saved views to help you organize your runs and filter out noise from worker nodes.
+
+To organize your distributed training runs:
+
+1. Set `job_type` for different node types to categorize your nodes based on their function:
+
+   ```python
+   # Main coordinating node
+   wandb.init(
+       project="distributed-training",
+       group="experiment_1",
+       job_type="main"
+   )
+
+   # Reporting worker nodes
+   wandb.init(
+       project="distributed-training", 
+       group="experiment_1",
+       job_type="worker"
+   )
```

Comment:
Let's highlight that each code snippet is in a different script and machine.

```suggestion
   ```python
   # Main coordinating node
   with wandb.init(project="distributed-training", group="experiment_1",job_type="main") as run:
       # Training code

   # Reporting worker nodes
   with wandb.init(project="distributed-training", group="experiment_1", job_type="worker") as run:
       # Training code
   ```

Reply:
^GitHub preview looks weird. Verify locally it works.

Reply:
Applied and verified locally.